templates/monitoring/document_sample_nonprod_v1/query.sql

```diff
@@ -0,0 +1,69 @@
+WITH extract_decoded AS (
+  SELECT
+    metadata.document_namespace,
+    metadata.document_type,
+    metadata.document_version,
+    '' AS error_message,
+    payload
+  FROM
+    `moz-fx-data-shar-nonprod-efed.payload_bytes_decoded.*`
+  WHERE
+    submission_timestamp
+> TIMESTAMP_SUB(current_timestamp, INTERVAL 1 day)
```

Comment:
This is strange formatting with the `>` completely unindented. Can we move this onto the previous line?

Reply:
This is a side-effect of the `script/format-sql` script. I can use the default BigQuery or Redash formatter here though.